Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
```

Comment:
### [crass](https://github.com/rgrove/crass/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.4 → 1.0.5

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rgrove/crass/compare/v1.0.4...v1.0.5).

These are the individual commits:
- (074e56f) [Update Travis test matrix](https://github.com/rgrove/crass/commit/074e56f2a9f10bb873fa8e708ef58a065d4281a2)
- (78d952e) [Remove 1.9.3 from the test matrix](https://github.com/rgrove/crass/commit/78d952e280b5919f104e88b77b633f4bb1a82ef6)
- (1608f38) [Remove test files and omit them](https://github.com/rgrove/crass/commit/1608f38ec19f3dad46e4a469039c62c5246cde4d)
- (30b9aa4) [Release 1.0.5](https://github.com/rgrove/crass/commit/30b9aa48458a5dd015fc86909c03707512cc3d96)


</details>